addons/web/static/tests/views/list_tests.js

```diff
@@ -59,11 +59,14 @@ QUnit.module('Views', {
                 ]
             },
             bar: {
-                fields: {},
+                fields: {
+                    foo: {string: "Foo", type: "char"},
+                    bar: {string: "Bar", type: "boolean"},
```

Comment:
I wouldn't use 'foo' and 'bar' here, because there are already data of type 'foo' and 'bar'.

Only case I would re-use these names for variables is in case of many2one or one2many.